network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixels.kt

```diff
@@ -377,6 +385,10 @@ class RealNetworkProtectionPixel @Inject constructor(
         firePixel(NETP_WAITLIST_BETA_ENABLED)
     }
 
+    override fun waitlistIncrementalRolloutTest() {
+        tryToFireDailyPixel(NETP_INCREMENTAL_ROLLOUT_TEST_PIXEL_DAILY)
```

Comment:
I am slightly confused about this. Isn't the spec that we send unique pixel sent once ever? How come we are send a daily pixel?

Reply:
NIT: Pixel contains appVersion